ui/app/adapters/pki/issuer.js

```diff
@@ -21,6 +21,22 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
     }
   }
 
+  createRecord(store, type, snapshot) {
+    const { record, adapterOptions } = snapshot;
+    let url = this.urlForQuery(record.backend);
+    if (adapterOptions.import) {
+      url = `${url}/import/bundle`;
```

Comment:
`issuers/import/bundle` accepts both certificate and keys, `issuers/import/cert` only accepts a certificate and errors if sent a key. After chatting with crypto, confirmed that UI should just use `/bundle`